src/plugins/table/table-utils.js

```diff
@@ -24,8 +40,23 @@ export const DESIGNER_CELL_ACTION = {
   disconnectCell: 'disconnect-cell'
 };
 
+export function calculateEvenColumnWidthsInPercent(columnCount) {
```

Comment:
If there's any risk `columnCount` could be zero, let's maybe default it to `1` here?

Reply:
Should never happen, but I added an if clause that'll return empty array in this case.